aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -654,6 +654,11 @@ Tensor& lgamma_out(Tensor& result, const Tensor& self) { return unary_op_impl_fl
 Tensor lgamma(const Tensor& self) { return unary_op_impl_float(self, lgamma_stub); }
 Tensor& lgamma_(Tensor& self) { return unary_op_impl_(self, at::lgamma_out); }
 
+// alias for lgamma, implements special.gammanln equivalent to
+// scipy.special.gammaln
+Tensor special_gammaln(const Tensor& self) { return self.lgamma(); }
```

Comment:
This is good for this PR because it illustrates the namespace working with a minimal change, but in the future let's have the existing torch.* operators alias their special counterparts (preferring the special namespace).